geomesa-features/geomesa-feature-avro/src/main/scala/org/locationtech/geomesa/features/avro/serialization/CollectionSerialization.scala

```diff
@@ -0,0 +1,299 @@
+/***********************************************************************
+ * Copyright (c) 2013-2022 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php. 
+ ***********************************************************************/
+
+package org.locationtech.geomesa.features.avro.serialization
+
+import java.nio.ByteBuffer
+import java.nio.charset.StandardCharsets
+import java.util.{Date, Locale, UUID}
+
+/**
+ * Serialization of lists and map types as opaque byte arrays, previously contained in AvroSimpleFeatureUtils
+ */
+object CollectionSerialization {
```

Comment:
this code was moved out of AvroSimpleFeatureUtils, probably doesn't need much/any review